components/CleanAction.coffee

```diff
@@ -0,0 +1,15 @@
+noflo = require 'noflo'
+
+exports.getComponent = ->
+  c = new noflo.Component
+  c.description = 'Strips state from action payloads for backwards compat'
```

Comment:
They don't care about additional attributes, but the old stores expect to have event payload as the payload of the `data` packet. Now it is in `data.payload` (to provide support for `data.state` as well). This component is there so we don't need to change how the old stores operate until we migrate them to modern middleware/reducer combos